.ci

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env groovy
+
+// https://github.com/apartmentlist/ci-shared-library
+@Library('ci-shared-library')_
+
+// Log Rotation
+properties([
+  buildDiscarder(
+    logRotator(
+      artifactDaysToKeepStr: '',
+      artifactNumToKeepStr: '',
+      daysToKeepStr: '30',
+      numToKeepStr: '100'
+    )
+  )
+]) //properties
+
+// Generate unique slave labels
+def k8s_label = "${UUID.randomUUID().toString()}"
+
+pipeline {
+  environment {
+    APP_NAME = 'bulk-processor'
+    APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE = 'true'
+    CI = 'true'
+    CLOUDSDK_CORE_DISABLE_PROMPTS = '1'
+    GIT_COMMIT_SHORT = sh(script: "git rev-parse --short ${GIT_COMMIT}", returnStdout: true).trim()
+    GIT_MESSAGE = sh(script: "git log --format=%B -n 1 ${GIT_COMMIT}", returnStdout: true).trim()
+    GIT_USER = sh(script: "git log -1 --pretty=format:'%ae'", returnStdout: true).trim()
+    GITHUB_URL = "https://github.com"
+    LANG = "en_US.UTF-8"
+    LANGUAGE = "en_US:en"
+    LC_ALL = "en_US.UTF-8"
+    PRODUCTION_DEPLOY="false"
+    SLACK_CHANNEL = "#staging-releases"
```

Comment:
because of the update on `post` this line won't be used as well. you can remove it.